internal/searcher/client.go

```diff
@@ -32,31 +32,40 @@ var cachedClient *client
 func Client() *client {
 	if cachedClient == nil {
 		endpoint, _ := url.JoinPath(searchHost(), "search")
+		// TODO: how to handle error
```

Comment:
just panic is probably fine since this error can't happen on the default host. It can only happen on a custom host. I guess we could also return an error here, but that's a bigger change.

Reply:
Then I think we can return a user error to warn users for invalid custom hosts?

Reply:
https://github.com/jetpack-io/devbox/pull/1173